### DISCUSSION TITLE
Scrape Targets with two Prometheus instances and Performance Impact on Filer

### DISCUSSION THREAD

**Discussion by jmg011** [NetApp/harvest#727] [Q&A] [answered]
Question: If I have two Prometheus Instances scraping same targets, will that cause any performance impact on filer?

Afaik, Harvest exporters exposes the metrics at the set interval, so if server can handle http requests from multiple Prometheus instances, we are good. Can someone confirm to this?

**Comment by rahulguptajss** [comment, marked as answer]
Yes, No impact on filer.

**Comment by jmg011** [comment]
@rahulguptajss Thanks for confirming. Another question with similar context, if I reduce the data poll duration from 3 minutes to 1 minute, there won't be any impact on filer, correct?

**Comment by rahulguptajss** [comment]
That means , you are doing zapi/rest calls more frequently which will have some impact on filer. How much? That may depend on muliple factors and tough to know without trying.

**Reply by jmg011** [reply]
Thank you @rahulguptajss Are there any benchmark numbers published for such use cases?
